kind-smoketest.sh

```diff
@@ -12,6 +12,9 @@ BIN_DIR="$(mktemp -d)"
 # kind binary will be here
 KIND="${BIN_DIR}/kind"
 
+CWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+KIND_CONFIG="${CWD}/kind-config.yaml"
```

Comment:
The kind-config should probably have a comment added that it's just a sample, we recommend most users use a single node cluster (the default) actually.